### PR TITLE
NEBULA-913 React Wrapper on Apollo Explorer

### PR DESCRIPTION
**Pull request by mayakoneval**
This PR 
1. straightens out the example files. We had an example pointing to the cdn and one pointing to the local files. We now have cdn examples and a single local example for local dev. 
2. splits the umd build and the cjs / esm build into 2 steps so we can continue to expose `EmbeddedExplorer` on window while exporting named variables.
3. updates the readme for the new react & vanilla js examples
4. adds a react component that embeds a given config's graph as well as adding a small react app as a local dev test
5. update circle config to run the umd build & upload it to the cdn
6. address [this issue](https://github.com/apollographql/embeddable-explorer/pull/12) by exposing a dispose function on `EmbeddedExplorer` class. We need this to clean up listeners when a react component is unmounted.
7. lint

Overall, this PR exports named vars `ApolloExplorer` which is the `EmbeddedExplorer` class, as well as `ApolloExplorerReact` which is a react component you can use to embed your graph.

### CODE REVIEW

**Comment by mayakoneval** [comment]
Requesting Jason, because I don't know if Chang has time to review this

**Review by cheapsteak — APPROVED** [review]
thanks so much, looks great 🙏 

one comment on ergonomics / hoping the react component can take full responsibility of finding the right DOM node so our users don't have to

File: src/react/index.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    return () => embed.dispose();
+  }, [props]);
+
+  return (
+    <div
+      className={props.className}
+      id={props.target?.substring(1) ?? DEFAULT_ELEMENT_ID}
+    />
+  );
+};
```
[the remark this answers is not in the document]
Comment (mayakoneval) [reply src/react/index.tsx:30]: Do you think folks will ever need to have access to this ref? Like if they want to select it by id or something? That was the only use case I had for target id as a string. I like this much better, just wondering if we should allow folks to pass in a ref at all

[the remark this answers is not in the document]
Comment (cheapsteak) [reply src/react/index.tsx:30]: 💭 oh hmm that's possible, probably doesn't hurt to accept a [forwardRef](https://reactjs.org/docs/forwarding-refs.html)

I believe the only change is to also assign the value of the element to the forwarded ref

```
export const ApolloExplorerReact = React.forwardRef((props, ref) => (
//...
    <div
      className={props.className}
      ref={(element) => {
        setWrapperElement(element);
        ref.current = element;
      }}
    />
```

**Pull request merged**